Components/Hydrator/ResourceHydrator/CustomerHydrator/PrivateCustomerHydrator.php

```diff
@@ -27,7 +27,7 @@ public function hydrateOrFetch(
         $shippingAddress = $data['shippingaddress'];
         $billingAddress  = $data['billingaddress'];
 
-        $phoneNumber = \preg_replace(self::PHONE_NUMBER_REGEX, '', $billingAddress['phone']);
+        $phoneNumber = \preg_replace(self::PHONE_NUMBER_REGEX, '', $billingAddress['phone'] ?? '');
```

Comment:
This seems to be missing in the changelog.

Reply:
This change was done during the compatibility for PHP 8 and implements a NULL check